ember-cli-build.js

```diff
@@ -37,5 +16,9 @@ module.exports = function(defaults) {
     behave. You most likely want to be modifying `./index.js` or app's build file
   */
 
+  app.import('vendor/highlight.pack.js', {
```

Comment:
For posterity's sake, what were the options used for generating this? Might be worth adding a comment above.